src/App.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisibleMovies] = useState<Movie[]>(moviesFromServer);
+
+  useEffect(() => {
+    const queryToCheck = query.toLowerCase();
+    const filtredMovies = moviesFromServer.filter(({ title, description }) => {
```

Comment:
will be better to move filtering inside set state function, because you want to set new state according to the current one